daemons/gptp/common/avbts_message.hpp

```diff
@@ -949,6 +949,8 @@ class PTPMessagePathDelayRespFollowUp:public PTPMessageCommon {
 	PortIdentity *requestingPortIdentity;
 
 	PTPMessagePathDelayRespFollowUp(void) { }
+
+	const static uint16_t SEQID_DIFF_THRESHOLD = 4;
```

Comment:
This doesn't appear to be used anymore within the diff; is it still needed?  I think there's a similar constant elsewhere that controls the timeout.